reana-ui/src/components/WorkflowProgressCircleBar.js

```diff
@@ -0,0 +1,103 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2024 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
```

Comment:
A functional observation. When I submit a BSM workflow, immediately after submission the progress is shown as:

![red](https://github.com/reanahub/reana-ui/assets/517546/b7bf1219-543e-40bc-9928-0d0e8661ed9a)

Here the "red" seems a bit harsh, so perhaps the colour is not appropriate.

Later when the engine starts and we know how many jobs are to be run, then the progress looks better:

![red2](https://github.com/reanahub/reana-ui/assets/517546/4524b837-a388-4c63-81bf-7f3853ef9637)

Perhaps we should show "grey" also in the first case?